make/testing_flags.mk

```diff
@@ -20,7 +20,7 @@ ifneq ($(pkg),)
 UNITPKG := $(PKG)/$(pkg)
 UNIT_TARGETED = yes
 COVER_PKG = $(PKG)/$(pkg)
-RACE_PKG = $(PKG)/$(pkg)
+RACE_PKG = echo $(PKG)/$(pkg)
```

Comment:
Was the echo needed here or a leftover?

Reply:
Needed, we need it so that we can pipe the result into the next bash command, since that's what we did with `GOLIST` as well.